.github/actions/run-gradle/action.yml

```diff
@@ -16,8 +16,8 @@
 # under the License.
 #
 ---
-name: "Gradle Setup"
-description: "Setup Java and Gradle"
+name: "Gradle Test Setup"
```

Comment:
This was a mis-copied name from our setup-gradle action. It should just be "Run Tests with Gradle" or similar.